aikau/src/test/resources/alfresco/services/ContentServiceTest.js

```diff
@@ -22,9 +22,9 @@
  * @author Dave Draper
  */
 define(["intern!object",
-        "intern/chai!assert",
-        "require",
-        "alfresco/TestCommon"],
+        "intern/chai!assert", 
+        "require", 
+        "alfresco/TestCommon"], 
```

Comment:
Trailing whitespace? We should decide whether to striping it or not (my IDE does, it seems yours doesn't).